vaadin-grid-pro-flow/src/main/java/com/vaadin/flow/component/gridpro/GridPro.java

```diff
@@ -96,6 +103,8 @@ private void setup() {
             EditColumn<E> column = (EditColumn<E>) this.idToColumnMap.get(e.getPath());
 
             column.getItemUpdater().accept(e.getItem(), e.getSourceItem().get(e.getPath()).asString());
+
+            getDataProvider().refreshAll();
```

Comment:
This is overkill. Use `refreshItem` instead.

Reply:
Updated.